.circleci/config.yml

```diff
@@ -395,3 +452,8 @@ workflows:
           filters:
             branches: { ignore: /.*/ }
             tags: { only: /.*/ }
+      - site_deploy_tag:
+          requires: [site_build]
```

Comment:
Need to add `site_build` job to the tag push workflow.
```
#!/bin/sh -eo pipefail
# site_deploy_tag: requires job \"site_build\" but \"site_build\" is not part of this workflow.
# The following jobs are unreachable: site_deploy_tag
```
from https://app.circleci.com/pipelines/github/massgov/mayflower/52/workflows/fabda5d4-b9d7-461e-88ae-bf63a1f0d17a/jobs/25838

Reply:
good catch!